### PR TITLE
Release embedded-hal{,-async,-nb} v1.0.0-rc.1, embedded-hal-bus v0.1.0-rc.1

### LINKED ISSUE #482: Update changelogs
There have been a number of changes that should have been added to the Changelogs.
See [here](https://github.com/rust-embedded/embedded-hal/compare/v1.0.0-alpha.11...HEAD)
Including, but not limited to:
- Removal of serial traits in `e-h`
- defmt support everywhere
- MSRV update everywhere
- `+ ?Sized` for blanket impls
- Move async `ExclusiveDevice` to `embedded-hal-bus`
- Merge of spi `ExclusiveDevice` in `embedded-hal-bus`
- async `DelayUs` implementation for `NoDelay`

I have not checked the `embedded-io*` changelogs since that ship has sailed.

### LINKED ISSUE #483: Document defmt feature
Regarding functionality and also that it is exempt of the stability guarantees of 1.0 releases.

### PR DESCRIPTION
**Pull request by Dirbaio**
Fixes #482 
Fixes #483

### CODE REVIEW

File: embedded-hal/CHANGELOG.md [hunk 1]
```diff
@@ -65,7 +71,7 @@ and this project adheres to [Semantic Versioning](http://semver.org/).
 *** This is (also) an alpha release with breaking changes (sorry) ***
 
 ### Changed
-- The Minimum Supported Rust Version (MSRV) is now 1.60.0
+- The Minimum Supported Rust Version (MSRV) is now 1.59.0
```
Comment (eldruin) [inline embedded-hal/CHANGELOG.md:74]:
```suggestion
- The Minimum Supported Rust Version (MSRV) is now 1.60.0
```

File: embedded-hal/CHANGELOG.md [hunk 2]
```diff
@@ -7,6 +7,12 @@ and this project adheres to [Semantic Versioning](http://semver.org/).
 
 ## [Unreleased]
 
+## [v1.0.0-rc.1] - 2023-08-15
+
+- The Minimum Supported Rust Version (MSRV) is now 1.60.0
+- Add optional `defmt 0.3` support.
```
Comment (eldruin) [inline embedded-hal/CHANGELOG.md:13]:
```suggestion
- Add optional `defmt` 0.3 support.
```

File: embedded-hal-bus/README.md [hunk 3]
```diff
@@ -28,9 +28,11 @@ This crate provides mechanisms to connect a `SpiBus` and a `SpiDevice`.
 In the case of I2C, the same `I2c` `embedded-hal` trait represents either an entire bus, or a device on a bus. This crate
 provides mechanisms to obtain multiple `I2c` instances out of a single `I2c` instance, sharing the bus.
 
-## Features
+## Optional Cargo features
 
-- `std`: enable shared bus implementations using `std::sync::Mutex`.
+- **`std`**: enable shared bus implementations using `std::sync::Mutex`.
+- **`async`**: enable `embedded-hal-async` support.
+- **`defmt-03`**: Derive `defmt::Format` from `defmt v0.3` for enums and structs.
```
Comment (eldruin) [inline embedded-hal-bus/README.md:35]:
```suggestion
- **`defmt-03`**: Derive `defmt::Format` from `defmt` 0.3 for enums and structs.
```

File: embedded-hal-bus/CHANGELOG.md [hunk 4]
```diff
@@ -7,6 +7,14 @@ and this project adheres to [Semantic Versioning](http://semver.org/).
 
 ## [Unreleased]
 
+## [v0.1.0-rc.1] - 2023-08-15
+
+- Updated `embedded-hal`, `embedded-hal-async` to version `1.0.0-rc.1`.
+- The Minimum Supported Rust Version (MSRV) is now 1.60.0
+- Added `embedded-hal-async` support to SPI `ExclusiveDevice`.
+- Added methods to access the inner bus to SPI `ExclusiveDevice`.
+- Add optional `defmt 0.3` support.
```
Comment (eldruin) [inline embedded-hal-bus/CHANGELOG.md:16]:
```suggestion
- Add optional `defmt` 0.3 support.
```

File: embedded-io-async/README.md [hunk 5]
```diff
@@ -10,6 +10,12 @@ This crate contains asynchronous versions of the [`embedded-io`](https://crates.
 
 This project is developed and maintained by the [HAL team](https://github.com/rust-embedded/wg#the-hal-team).
 
+## Optional Cargo features
+
+- **`std`**: Adds `From` impls to convert to/from `std::io` structs, adds `std::error::Error` impls.
+- **`alloc`**: Adds blanket impls for `Box`, adds `Write` impl to `Vec`.
+- **`defmt-03`**: Derive `defmt::Format` from `defmt v0.3` for enums and structs.
```
Comment (eldruin) [inline embedded-io-async/README.md:17]:
```suggestion
- **`defmt-03`**: Derive `defmt::Format` from `defmt` 0.3 for enums and structs.
```

File: embedded-io/README.md [hunk 6]
```diff
@@ -19,6 +19,12 @@ while avoiding `dyn` or `Box`. This is consistent with how errors are handled in
 - In `std::io`, the `Read`/`Write` traits might be blocking or non-blocking (i.e. returning `WouldBlock` errors) depending on the file descriptor's mode, which is only known at run-time. This allows passing a non-blocking stream to code that expects a blocking
 stream, causing unexpected errors. To solve this, `embedded-io` specifies `Read`/`Write` are always blocking, and adds new `ReadReady`/`WriteReady` traits to allow using streams in a non-blocking way.
 
+## Optional Cargo features
+
+- **`std`**: Adds `From` impls to convert to/from `std::io` structs, adds `std::error::Error` impls.
+- **`alloc`**: Adds blanket impls for `Box`, adds `Write` impl to `Vec`.
+- **`defmt-03`**: Derive `defmt::Format` from `defmt v0.3` for enums and structs.
```
Comment (eldruin) [inline embedded-io/README.md:26]:
```suggestion
- **`defmt-03`**: Derive `defmt::Format` from `defmt` 0.3 for enums and structs.
```

**Review by eldruin — COMMENTED** [review]
Thank you!
Just a couple of nitpicks. I do not care about the formatting of the `defmt` 0.3 support but let's keep it consistent

File: embedded-hal/CHANGELOG.md (hunk 1 above)
Comment (Dirbaio) [reply embedded-hal/CHANGELOG.md:74]: This is correct, I had inadvertently changed this with a `1.59`->`1.60` mass replace.

Comment (eldruin) [reply embedded-hal/CHANGELOG.md:74]: I see, thanks!

**Review by eldruin — APPROVED** [review]
Thank you!

**Pull request merged**